OpenSim/Tools/Test/sphere.vtp

```diff
@@ -0,0 +1,883 @@
+<?xml version="1.0"?>
+<VTKFile type="PolyData" version="0.1" byte_order="LittleEndian" compressor="vtkZLibDataCompressor">
```

Comment:
The first thing I usually do to debug is to visualize! So if it is immediately (visually) apparent that a transform is off it saves time - I think. Otherwise seeing spheres I would just be scratching my head and would have to diff the visualization text files and also factor in a possible change in order, etc...